pandas/core/groupby/generic.py

```diff
@@ -1074,15 +1074,8 @@ def _aggregate_frame(self, func, *args, **kwargs):
         else:
             for name in self.indices:
                 data = self.get_group(name, obj=obj)
-                try:
-                    fres = func(data, *args, **kwargs)
-                except AssertionError:
-                    raise
-                except Exception:
-                    wrapper = lambda x: func(x, *args, **kwargs)
-                    result[name] = data.apply(wrapper, axis=axis)
```

Comment:
Why is this fallback not needed? It was never reached according to the tests? Or is it done elsewhere?